lib/Runtime/Library/JavascriptLibrary.h

```diff
@@ -1216,8 +1216,9 @@ namespace Js
 
         JavascriptFunction* EnsureArrayPrototypeValuesFunction();
         
-
     public:
+        Var GetArrayPrototypeBuiltinIterator() { return EnsureArrayPrototypeValuesFunction(); }
```

Comment:
There is already a GetArrayPrototypeValuesFunction() method defined earlier in this class.  However it does not do the ensure call.  It should be updated to do the ensure and then you can use it instead of creating a new method.